src/amo/withExperiment.js

```diff
@@ -305,15 +300,14 @@ export const withExperiment =
       render() {
         const { _isExperimentEnabled, ...props } = this.props;
 
+        const { variant } = this.experimentSetup(this.props);
```

Comment:
Gaaa, we should not have that here because `experimentSetup` could dispatch actions it seems, and that is not allowed in `render()`. The local dev env is broken for this reason right now (not sure why yet, though).